.github/workflows/pr-check.yaml

```diff
@@ -84,3 +84,91 @@ jobs:
+          curl -L "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_22.04/Release.key" | sudo apt-key add -
+          sudo apt-get update -qq
+          sudo apt-get -qq -y install podman || { echo "Start fallback steps for podman nightly installation from a static mirror" && \
+            sudo sh -c "echo 'deb http://ftp.lysator.liu.se/pub/opensuse/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_22.04/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:unstable.list" && \
+            curl -L "http://ftp.lysator.liu.se/pub/opensuse/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_22.04/Release.key" | sudo apt-key add - && \
+            sudo apt-get update && \
+            sudo apt-get -y install podman; }
+          podman version
+          # downgrade conmon package version to workaround issue with starting containers, see  https://github.com/containers/conmon/issues/475
+          # remove once the repository contains conmon 2.1.10
+          wget https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_22.04/amd64/conmon_2.1.2~0_amd64.deb -O /tmp/conmon_2.1.2.deb
+          sudo apt install /tmp/conmon_2.1.2.deb
+
+      - name: Build Podman Desktop for E2E tests
+        working-directory: ./podman-desktop
+        run: |
+          yarn --frozen-lockfile
+          yarn test:e2e:build
+
+      - name: Get yarn cache directory path
+        working-directory: ./podman-desktop-extension-bootc
+        id: yarn-cache-dir-path
+        run: echo "dir=$(yarn cache dir)" >> ${GITHUB_OUTPUT}
+
+      - uses: actions/cache@v4
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Ensure getting current HEAD version of the test framework
+        working-directory: ./podman-desktop-extension-bootc/tests/playwright
+        run: yarn add -D @podman-desktop/tests-playwright@next 
+
+      - name: Execute yarn in Bootc Extension
+        working-directory: ./podman-desktop-extension-bootc
+        run: yarn --frozen-lockfile
+      
+      - name: Build bootc extension from container file
+        working-directory: ./podman-desktop-extension-bootc
+        run: |
+          yarn build
+          podman build -t local_bootc_image ./
+          CONTAINER_ID=$(podman create localhost/local_bootc_image)
+          podman export $CONTAINER_ID > /tmp/local_bootc_extension.tar
+          sudo cp /tmp/local_bootc_extension.tar ~/.local/share/containers/podman-desktop/plugins
+          cd ~/.local/share/containers/podman-desktop/plugins/
+          tar -xvf local_bootc_extension.tar
```

Comment:
use `-C ~/.local/share/containers/podman-desktop/plugins/` to tar command ?

Reply:
Just the location needs to be one used by e2e tests. This is for podman installed locally. E2E tests use different location. The `-C` flag is correct though!